templates/Dockerfile-deb.tmpl

```diff
@@ -83,13 +89,36 @@ RUN curl -sL https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
     && chmod +x /usr/bin/jq
 
 # install packagecloud repos for pg_auto_failover
-RUN curl -s https://packagecloud.io/install/repositories/citusdata/community/script.deb.sh | bash \
-    && rm -rf /var/lib/apt/lists/*
+RUN curl -s https://packagecloud.io/install/repositories/citusdata/community/script.deb.sh | bash
 
 # patch pg_buildext to use multiple processors
 COPY make_pg_buildext_parallel.patch /
 RUN patch `which pg_buildext` < /make_pg_buildext_parallel.patch
 
+# install cmake dependencies and pgazure build dependencies
+RUN apt install -y  checkinstall \
+                    git \
+                    libtemplate-perl \
+                    make \
+                    pkg-config \
+                    tar \
+                    unzip \
+                    uuid \
+                    uuid-dev \
+                    zip \
```

Comment:
these should be made part of the other install list

Reply:
Merged with upper block